Microsoft.Toolkit.Uwp.UI.Controls/RangeSelector/RangeSelector.cs

```diff
@@ -35,6 +36,7 @@ public partial class RangeSelector : Control
         private const double DefaultMinimum = 0.0;
         private const double DefaultMaximum = 1.0;
         private const double DefaultStepFrequency = 1;
+        private static readonly TimeSpan TimeToHideToolTipOnKeyUp = TimeSpan.FromSeconds(1);
```

Comment:
Is the name fine here?
Also is the amount of time fine? It felt okay from my testing.

Reply:
Seemed a smidge long, but I think that's fine, better safe than sorry. 🙂